remoteappmanager/static/js/home/views/application_view.js

```diff
@@ -42,7 +42,15 @@ define([
             '          </ul>' +
 
             '          <h4>Configuration</h4>' +
-            '          <form class="configuration"><fieldset></fieldset></form>' +
+            '          <form class="configuration">' +
+            '            <fieldset v-if="current_app.configurables.length === 0">No configurable options for this image</fieldset>' +
+            '            <fieldset v-else :disabled="current_app.is_starting()">' +
+            '              <component v-for="configurable in current_app.configurables"' +
+            '                         :is="configurable.tag"' +
+            '                         :value="configurable.value"' +
```

Comment:
The `.sync`doesn't work like expected... The child `selected_value` is not correctly synchronized with the parent value.